packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -322,6 +323,9 @@ class ExpansionTile extends StatefulWidget {
   /// may replace the rotating expansion arrow icon.
   final Widget? trailing;
 
+  /// Specifies if the [ExpansionTile] should build a default trailing icon if [trailing] is null.
```

Comment:
The comment should probably focus on the "show/no show" aspect, since that's the name of the parameter.

Maybe something like this instead?

```suggestion
  /// If true, the [ExpansionTile] will show a trailing icon.
  ///
  /// If true and [trailing] is set, [ExpansionTile] will use that trailing icon,
  /// otherwise a default trailing icon will be created.
```

Reply:
This was/is actually a valid change-request.
> "Specifies if the [ExpansionTile] should build a default trailing icon if [trailing] is null."

This current definition is somehow misleading, as this property simply toggles visibility of the trailing icon, be it the user-defined `trailing` or the default.
I personally got confused and had to jump into the source code to better understand what was happening.